package/AUTHORS

```diff
@@ -151,7 +151,8 @@ Chronological list of authors
   - Nicholas Craven
   - Mieczyslaw Torchala
   - Haochuan Chen
-
+2021
+  - Aditya Kamath
```

Comment:
Could you just make sure there's an empty line after your name and before `External code`?